src/core/lib/security/transport/client_auth_filter.cc

```diff
@@ -248,9 +256,11 @@ static void send_security_metadata(grpc_call_element* elem,
   }
 
   if (channel_call_creds != nullptr && call_creds_has_md) {
+    // When compositing credentials, use the call credenttials first and then
```

Comment:
This seems wrong to me.  The order should not matter.  And if it does, then changing this is not safe, because it could break things for existing users.